src/gtimelog/main.py

```diff
@@ -1358,11 +1381,13 @@ def __init__(self):
         self.connect('notify::showing-today', self.queue_update)
         self.connect('notify::detail-level', self.queue_update)
         self.connect('notify::time-range', self.queue_update)
+        self.connect('notify::log-order', self.queue_update)
         self.connect('notify::hours', self.queue_footer_update)
         self.connect('notify::office-hours', self.queue_footer_update)
         self.connect('notify::current-task', self.queue_footer_update)
         self.connect('notify::now', self.queue_footer_update)
         self.connect('notify::filter-text', self.queue_update)
+        self.connect('notify::tasks', self.queue_update)
```

Comment:
For this to work you have to define a property `tasks` in this class (which is LogView, right?)  See how class TaskListView does it:

```python
    tasks = GObject.Property(
        type=object, nick='Tasks',
        blurb='The task list (an instance of TaskList)')
```

And then instead of a `self.log_view.set_tasks(tasks)`, bind the property in `Window.__init__`, where it currently binds Window.tasks to TaskListView.tasks:

```python
        self.bind_property('tasks', self.task_list, 'tasks', GObject.BindingFlags.DEFAULT)    # <-- existing code
        self.bind_property('tasks', self.log_view, 'tasks', GObject.BindingFlags.DEFAULT)    # <-- new code
```

Then the assignment to `self.tasks` in Window.load_tasks will automatically invoke `LogView.queue_update`.

Reply:
Aha, I couldn't find any conclusive documentation. Thanks for the explanations. Implemented and works.